jobs/scripts/parallel-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+pip install -r requirements.txt
+buildout buildout:git-clone-depth=50 -c py3.cfg
+
+export PATH="/usr/lib/chromium-browser:$PATH"
+export ROBOT_BROWSER=chrome
+
+xvfb-run -a --server-args='-screen 0 1920x1200x24' bin/test --all -j 6 --xml
```

Comment:
Cross-posting from https://github.com/plone/jenkins.plone.org/pull/244

We'll rather split this so that we'll make the non-robot ones parallel first.

Also, can the `-j` flag get its parallelity from `{weight}`? This way we could use `-j` everywhere and set up `{weight}` 1 for everything and then bump things up one by one where we see they already run.